cluster/examples/kubernetes/ceph/cluster-on-local-pvc.yaml

```
@@ -231,6 +231,8 @@ spec:
               volumeMode: Block
               accessModes:
                 - ReadWriteOnce
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement
```

This update would be needed in several places
```suggestion
    # when onlyApplySpecificOSDPlacement is false, will merge both placement.All() and storageClassDeviceSets.Placement
```